tests/test_pagination.py

```diff
@@ -331,6 +348,62 @@ def test_relay_cursor_sorted(client, data):
     }
 
 
+def test_relay_cursor_sorted_none_desc(client, data_with_nulls):
```

Comment:
I know we don't generally have any tests  like this, but it would (imo) be good to have a few tests around the expected SQL output, both to ensure that it's correct, and to prevent subtle regressions